sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.processing.service.ts

```diff
@@ -49,23 +60,23 @@ export class IER12ProcessingService {
     }
     this.logger.log(`Found ${pendingAssessments.length} assessments.`);
     const fileRecords: Record<string, IER12Record[]> = {};
-    pendingAssessments.forEach((pendingAssessment) => {
+    for (const assessment of pendingAssessments) {
```

Comment:
just wondering the reason for this change

Reply:
I changed the code from forEach to for as I was doing sync call inside the loop. Then I removed the async call but just left is back to be for.